pkg/app/pipedv1/plugin/azure/config/func_config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+type AzureDeployTemplate struct {
+	DeploymentName          string `json:"deploymentName"`
+	DeploymentTemplateFile  string `json:"deploymentTemplateFile"`
+	DeploymentParameterFile string `json:"deploymentParameterFile"`
```

Comment:
[nit][IMO] SInce `AzureDeployTemplate` contains `Deploy`, I think each "Deployment" is not necessary

```suggestion
	Name          string `json:"name"`
	TemplateFile  string `json:"templateFile"`
	ParameterFile string `json:"parameterFile"`
```